pytuya/__init__.py

```diff
@@ -125,7 +125,7 @@ def hex2bin(x):
 }
 
 class XenonDevice(object):
-    def __init__(self, dev_id, address, local_key=None, dev_type=None):
+    def __init__(self, dev_id, address, local_key=None, dev_type=None, connection_timeout=10):
```

Comment:
I'm wondering is the default should be zero, i.e. no-timeout. I need to think about it, going to merge as-is with 10 secs as the default.